prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -16,6 +16,7 @@
 
 
 class DeployableKnownOutcomeAgent(DeployableTraderAgent):
+    bet_on_n_markets_per_run = 20
     model = "gpt-4-1106-preview"
```

Comment:
@evangriffiths gpt-4 is the most expensive, but it's known outcome that's a most profitable, hard choice, wdyt?